public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPrevious.tsx

```diff
@@ -14,11 +16,15 @@ export interface ReturnToPreviousProps {
 
 export const ReturnToPrevious = ({ href, title }: ReturnToPreviousProps) => {
   const styles = useStyles2(getStyles);
+  const { chrome } = useGrafana();
+  const cleanSessionStorage = () => window.sessionStorage.removeItem('returnToPrevious');
   const handleOnClick = () => {
-    console.log('Going to...', href);
+    href && locationService.push(href);
```

Comment:
do we need the `href` check here? it's always defined. or are we explicitly filtering out `''`?